mollie-payments-for-woocommerce.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * Plugin Name: Mollie Payments for WooCommerce
+ * Plugin URI: https://www.mollie.com
+ * Description: Accept payments in WooCommerce with the official Mollie plugin
+ * Version: 5.1.8
+ * Author: Mollie
+ * Author URI: https://www.mollie.com
+ * Requires at least: 3.8
+ * Tested up to: 5.1
```

Comment:
Tested up to : 5.2

Reply:
See above. Also, this is probably something that should be addressed in a `release/*` branch.